.gitlab-ci.yml

```diff
@@ -193,13 +202,11 @@ build-linux-release:               &build
     - cp -r scripts/docker/* ./artifacts
     - sccache -s
 
-
-
-
 .publish-build:                    &publish-build
   stage:                           publish
   dependencies:
     - build-linux-release
+    - build-wasm-release
```

Comment:
can artifacts be fetched that way from two different jobs which ran in parallel before?

Reply:
Yep, wasn't sure about this myself so I made a test job to verify it: https://gitlab.parity.io/parity/polkadot/-/jobs/507196

Reply:
nice job!